native/src/App.tsx

```diff
@@ -21,9 +21,7 @@ import SnackbarContainer from './components/SnackbarContainer'
 import NetInfo from '@react-native-community/netinfo'
 import sendTrackingSignal from './services/sendTrackingSignal'
 import useSendOfflineJpalSignals from './hooks/useSendOfflineJpalSignals'
-import { enableScreens } from 'react-native-screens'
 
-enableScreens(true)
```

Comment:
I think it would be good tp explicitly state false here. Maybe screens gets turned on by default at some point in time.